js/components/select.tsx

```diff
@@ -421,13 +450,25 @@ class Select extends React.Component<IItemsComponentProps, IDropDownState> {
             widthClassName = 'width-specified';
         }
 
+        // Capturing the clear button presses as they are skipped by the component
+        var clearSelection = (e) => {
+            if ($(e.target).closest('.react-selectize-reset-button-container').length > 0) {
+                this.props.clearSelection();
+            }
+        };
```

Comment:
Is this still required now the dropdown will be inverted? Won't the input stop jumping?

Reply:
I'l test it, I think I should have gotten rid of this you're right

Reply:
Removed because it works as you said.